react-common/components/share/ShareInfo.tsx

```diff
@@ -264,6 +277,12 @@ export const ShareInfo = (props: ShareInfoProps) => {
                                         leftIcon={"icon share"}
                                         onClick={handleDeviceShareClick}
                                     />}
+                                    {pxt.appTarget?.appTheme?.multiplayerShareButton && <Button className="square-button multiplayer-host"
+                                        title={lf("Start a multiplayer lobby")}
```

Comment:
Would "Host a multiplayer session" maybe be clearer? Or perhaps "Play in multiplayer"?

Reply:
Hm, that's a good question, all three sound good to me / I'm sure I've seen each before in full games -- @unthinkmedia could I get your opinion on this?